components/status/StatusLink.vue

```diff
@@ -16,6 +18,13 @@ function onclick(evt: MouseEvent | KeyboardEvent) {
   const text = window.getSelection()?.toString()
   if (!el && !text)
     go(evt)
```

Comment:
Is it worth wondering if it would be safer to place the new analytics before this `go`, in case anchor tags were changed to redirect on the current tab instead of a new one?

This train of thought stems from [login/signup button clicks intermittently failing](https://github.com/MozillaSocial/elk/pull/64#issuecomment-1767049244) to register before they do redirects

Reply:
Were you able to discern when the `go` function gets called? I haven't been able to figure it out.

Reply:
@wtfluckey  Yup. When you click on a post in a feed and it takes you to the url for the standalone post.